### DISCUSSION TITLE
read from provider(s)

### DISCUSSION THREAD

**Discussion by Fanom2813** [nylo-core/nylo#201] [General]
Hello this project is great , i am new and i would like to know how to read / get data from one or many providers ?!

thanks!

**Comment by agordn52** [comment]
Hi @Fanom2813,

Thanks for your feedback!

Re. Providers
I'd recommend reading the [docs](https://nylo.dev/docs/6.x/providers) to learn what 'Providers' are in Nylo.

> In Nylo, providers are booted initially from your main.dart file when your application runs. All your providers reside in /lib/app/providers/*, you can modify these files or create your providers using [Metro](https://nylo.dev/docs/6.x/metro#make-provider). Providers can be used when you need to initialize a class, package or create something before the app initially loads. I.e. the route_provider.dart class is responsible for adding all the routes to Nylo.
